subsys/emds/Kconfig

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig EMDS
+	bool "Emergency Data Storage [EXPERIMENTAL]"
+	select EXPERIMENTAL
+	depends on PARTITION_MANAGER_ENABLED
+	depends on FLASH_MAP
+	default n
+	help
+	  Enable Emergency Data Storage
+
+if EMDS
+
+config EMDS_SECTOR_COUNT
+	int "Sector count of the emergency data storage area"
+	default 1
+	help
+	  Number of sectors used for the emergency data storage area
+
+config EMDS_THREAD_STACK_SIZE
+	int "Stack size for the emergency data storage thread"
+	default 500
+	help
+	  Size of the stack that is initialized by the emergency data storage
+
+config EMDS_THREAD_PRIORITY
+	int "Priority for the emergency data storage thread"
+	default 0
+	help
+	  Cooperative priority for the emergency data storage thread. This will
+	  be used through K_PRIO_COOP(x), that means higher value gives lower
+	  priority.
+
+config EMDS_FLASH_TIME_WRITE_ONE_WORD_US
+	int "Time to write one word into flash"
+	default 41
```

Comment:
It is best to move this to emds header. This should be choosen automatically depending upon the selected chip. This value is not the same for chips. Using incorrect value here will give wrong time estimates.

Reply:
I disagree based on the previous comment. And I do not want to make this library be dependent on some board type h file. The user should find these values in the datasheet and then do measurment for them self. So I would like this to be set for each project by the user, without having to change any code.

Reply:
Ok. Can you at least mention the source of this value, and update the description to tell the user where to find this?

Reply:
Added where to find this in documentation.